cmd/openshift-install/create.go

```diff
@@ -638,3 +646,9 @@ The cluster should be accessible for troubleshooting as detailed in the document
 https://docs.openshift.com/container-platform/latest/support/troubleshooting/troubleshooting-installations.html
 The 'wait-for install-complete' subcommand can then be used to continue the installation`)
 }
+
+func checkIfAgentCommand(assetStore asset.Store) {
```

Comment:
Would you need the checkIfAgentCommand() at multiple locations? If so, the warning msg can be used outside this method to then display context specific msg and the method can just indicate if it is an agent install or not.